irohad/torii/torii_service_handler.hpp

```diff
@@ -80,22 +81,30 @@ namespace torii {
      * &CommandService::AsyncService::RequestTorii
      * @param rpcHandler - handler of rpc in ServiceHandler.
      */
-    template <typename AsyncService, typename RequestType,
+    template <typename AsyncService,
+              typename RequestType,
               typename ResponseType>
-    void enqueueRequest(
-        network::RequestMethod<AsyncService, RequestType, ResponseType>
-            requester,
-        network::RpcHandler<ToriiServiceHandler, AsyncService, RequestType,
-                            ResponseType>
-            rpcHandler,
-        AsyncService& asyncService) {
+    network::Call<ToriiServiceHandler, AsyncService, RequestType, ResponseType>
```

Comment:
Same issue with auto as before.

Reply:
Can't use auto here because of nullptr (